R/serializer-html.R

```diff
@@ -1,7 +1,12 @@
 #' @rdname serializers
 #' @export
-serializer_html <- function(){
-  function(val, req, res, errorHandler){
+serializer_html <- function(...) {
```

Comment:
serialize_null should accept arguments. check all other serializers accept arguments and throw if any are provided